Checking.rb

```diff
@@ -0,0 +1,43 @@
+module Bank
+  # this will inherit behavior from Account class with '<'
+  class CheckingAccount < Account
+
+    attr_reader :checks
```

Comment:
Are you using this attribute to tie to the `@checks_used` variable anywhere or is this separate?